session5/src/homework/EqualityEx.java

```diff
@@ -0,0 +1,17 @@
+package homework;
+
+public class EqualityEx {
+    public static void main(String[] args) {
+
+        Integer firstNumber = 10;
+        Integer secondNumber = 20;
+
+        System.out.println( firstNumber == secondNumber);
+
+        String firstString = "10";
+        String secondString = "20";
```

Comment:
Cool! :) but here is an extra challenge: instead of hard-coding the values of these two Strings with "10" and "20", can you convert `firstNumber` and `secondNumber` to Strings? :D 
Hint: there are 2 ways to do it, one of them is using a static helper method from `String` class.